acceptance/tests/load_libfacter.rb

```diff
@@ -1,31 +1,32 @@
-test_name 'Ruby can load libfacter'
+test_name 'C100161: Ruby can load libfacter' do
+  tag 'risk:high'
 
-agents.each do |agent|
-  on agent, "env PATH=\"#{agent['privatebindir']}:${PATH}\" which ruby"
-  ruby_path = stdout.chomp
+  agents.each do |agent|
+    ruby_path = on(agent, "env PATH=\"#{agent['privatebindir']}:${PATH}\" which ruby").stdout.chomp
```

Comment:
There is a helper method available for this. It should be used rather than duplicating the code.
https://github.com/puppetlabs/facter/blob/master/acceptance/lib/puppet/acceptance/common_utils.rb#L4